pages/faq/troubleshooting/troubleshooting.md

```diff
@@ -90,13 +96,40 @@ If you have a screen attached to your Raspberry Pi and notice that there is a sm
+
+Starting with L4T 32.1 the NVidia partition layout has changed, so did the firmwares and bootloaders stored on the raw partitions. The hostOS update process thus needs to re-create all the NVidia partitions that were modified in the newer L4T, to allow for the newer bootloaders and kernels to run. L4T 28.X partition layout is not compatible with the L4T 32.X format and vice-versa, which means that a newer kernel cannot boot with older partition format, nor can an older kernel boot with the new layout and contents. Therefore, the hostOS update process should not interrupted on the Jetson family of boards, otherwise the device may not boot. Please ensure the device is powered correctly or has sufficient battery, the internet connectivity is stable and that it is not powered-off or rebooted manually during the hostOS update.
+
+In the case of a device that will not boot because of an interrupted update procedure, it will need to be re-flashed. This can be done by [downloading the L4T Driver Package][l4t-download] (BSP), unpacking it, putting the device in recovery mode and executing `sudo ./flash.sh jetson-tx2 mmcblk0p1`. Once the Nvidia partitions are finished writing, the device can be booted using the appropriate BalenaOS flasher SD-CARD image for the targeted L4T version.
+
+## NVidia Jetson Nano SD-CARD and Jetson Nano 2GB
+
+### Unable to boot BalenaOS image downloaded from Balena cloud
+
+Starting with L4T 32.5, all bootloaders and firmwares on the Jetson Nano family of boards have been moved from the sd-card/eMMC to the device's QSPI flash memory. L4T 32.5 and newer bootloaders are not compatible with previous L4T releases.
+
+If you purchased a Jetson Nano SD-CARD device that won't boot BalenaOS but boots the NVidia Ubunt image, or if you used an NVidia 32.5 or newer Ubuntu image with your Jetson Nano and wish to run BalenaOS, please first [download][l4t-download] the L4T 32.4.4 BSP archive for Jetson Nano, unpack it and flash the device's QSPI memory by executing `sudo ./flash.sh jetson-nano-qspi-sd mmcblk0p1`. You can then boot BalenaOS v2.75.0+rev1 which is based on L4T 32.4.4. You can then switch to a newer L4T version by performing a hostOS update from the dashboard panel. You can always see what L4T version the device runs by issuing `uname -a`.
+
+IMPORTANT: On Jetson devices it is important to not interrupt the HostOS update process. Please ensure the device is powered correctly or has sufficient battery, the internet connectivity is stable and that the device is not rebooted or shutdown during the hostOS update, otherwise the device may fail to boot and will need to be re-flashed.
+
+## NVidia Jetson L4T versions
+
+### How can I determine which L4T a BalenaOS image uses?
+
+If the device is running, typing `uname -a` will show the L4T in the kernel version string. Aditionally, the L4T version is visible in the [balena-jetson github repository][tegra-binaries-version] for every release tag.
```

Comment:
```suggestion
If the device is running, typing `uname -a` will show the L4T in the kernel version string. Additionally, the L4T version is visible in the [balena-jetson github repository][tegra-binaries-version] for every release tag.
```

Reply:
Thanks for spotting those typos @klutchell, I updated the PR